### PR TITLE
phase 2 upload

### PR DESCRIPTION
**Pull request by John-A-Davies**
Changes to enable editing of z/OS datasets.

### CODE REVIEW

**Review by 1000TurquoisePogs — COMMENTED** [review]
This looks good, but also looks subject to change on what ends up happening with ZSS

**Comment by John-A-Davies** [comment]
References to sessionID are no longer required, as this was a tryout solution which is now superseded.

File: bootstrap/src/uri/dsm-uri.ts [hunk 1]
```diff
@@ -34,9 +34,14 @@ export class DsmUri implements ZLUX.UriBroker {
   omvsSegmentUri(): string {
     return `${this.serverRootUri('omvs')}`;
   }
+
+
```
Comment (DivergentEuropeans) [inline bootstrap/src/uri/dsm-uri.ts:38]: line breaks

File: bootstrap/src/uri/mvd-uri.ts [hunk 2]
```diff
@@ -54,9 +54,15 @@ export class MvdUri implements ZLUX.UriBroker {
   omvsSegmentUri(): string {
     return `${this.serverRootUri('omvs')}`;
   }
+
+
```
Comment (DivergentEuropeans) [inline bootstrap/src/uri/mvd-uri.ts:58]: line breaks

File: virtual-desktop/package.json [hunk 3]
```diff
@@ -13,7 +13,9 @@
     "i18n": "ng-xi18n -p tsconfig.ngx-i18n.json --i18nFormat=xlf  --outFile=messages.xlf && xliffmerge -p xliffmerge.json",
     "lint": "tslint -c tslint.json \"src/**/*.ts\""
   },
-  "dependencies": {},
+  "dependencies": {
+    "@types/qs": "^6.9.3"
+  },
```
Comment (DivergentEuropeans) [inline virtual-desktop/package.json:18]: Do we need these other 2 npm packages?

**Review by DivergentEuropeans — COMMENTED** [review]
1 question, otherwise lgtm

File: virtual-desktop/package.json (hunk 3 above)
Comment (John-A-Davies) [reply virtual-desktop/package.json:18]: Removed.

**Pull request closed without merging**